src/components/Filters/Filters.types.ts

```diff
@@ -0,0 +1,13 @@
+import { Option } from './Select/Select.types';
+
+export interface FilterRow {
+  uuid: string;
+  conditionOptions: Option[];
+  dataOptions: Option[];
```

Comment:
Hmm I think that `dataOption` should be shared across all filter rows. As it should represent columns in table. Not sure if filters are used without the datatable does make sense to have different data points for different rows.

Reply:
It makes sense to move it one level up